tests/test_grpc.py

```diff
@@ -373,3 +374,31 @@ def test_end_training_denied(  # pylint: disable=unused-argument
     with pytest.raises(grpc.RpcError):
         reply = participant_stub.EndTraining(coordinator_pb2.EndTrainingRequest())
         assert reply.status_code == grpc.StatusCode.PERMISSION_DENIED
+
+
+@pytest.mark.integration
+def test_start_participant(mock_coordinator_service):
+    """[summary]
+
+    .. todo:: Advance docstrings (https://xainag.atlassian.net/browse/XP-425)
+
+    Args:
+        mock_coordinator_service ([type]): [description]
+    """
+
+    init_weight = [np.arange(10), np.arange(10, 20)]
+    mock_coordinator_service.coordinator.weights = init_weight
+
+    # mock a local participant with a constant train_round function
+    with mock.patch("xain_sdk.participant.Participant") as mock_obj:
+        mock_local_part = mock_obj.return_value
+        mock_local_part.train_round.return_value = init_weight, 1, {}
+
+        start_participant(mock_local_part, "localhost:50051")
+
+        coord = mock_coordinator_service.coordinator
+        assert coord.state == coordinator_pb2.State.FINISHED
+        # coordinator set to 2 round for good measure, but the resulting
+        # aggregated weights are the same as a single round
+        assert coord.current_round == 2
+        np.testing.assert_equal(coord.weights, [np.arange(start=10, stop=29, step=2)])
```

Comment:
maybe can you please specify in the comment that we expect the initial weights to have been summed together? it was not extra obvious